src/OrchardCore.Modules/OrchardCore.Workflows/Services/WorkflowManager.cs

```diff
@@ -150,6 +153,13 @@ public async Task TriggerEventAsync(string name, IDictionary<string, object> inp
                     continue;
                 }
 
+                // Check the max recursion depth of workflow executions.
+                if (_maxRecursionDepth > MaxRecursionDepth)
+                {
+                    _logger.LogError("The max recursion depth of 'Workflow' executions has been reached");
```

Comment:
```suggestion
                    _logger.LogError("The max recursion depth of 'Workflow' executions has been reached.");
```

Also for the other log message.

Reply:
Okay